README.md

```diff
@@ -78,11 +78,50 @@ To use in Django project install with extra dependencies:
 pip install pyuploadcare[django]
 ```
 
-## Usage example
+## Usage examples
+
+After package [installation](#requirements-and-installation), you’ll need API keys: public and secret. Get them in [Uploadcare dashboard](https://app.uploadcare.com/projects/-/api-keys). If you don’t have an account yet, you can use demo keys, as in example. However, the files on demo account are regularly removed, so create an account as soon as Uploadcare catches your fancy.
+
+In these examples we’re going to use the aforementioned demo keys.
 
 ### Basic usage
 
-<!-- TODO: simple upload via API request / and maybe something more-->
+Let’s start with the basics. Say, you want to upload a file:
+
+```python
+from pyuploadcare import Uploadcare
+
+uploadcare = Uploadcare(public_key="demopublickey", secret_key="demoprivatekey")
+with open("sample-file.jpeg", "rb") as file_object:
+    ucare_file = uploadcare.upload(file_object)
+```
+
+And your file is now uploaded to the Uploadcare CDN. But how do you access it from the web? It’s really simple:
+
+```python
+print(ucare_file.cdn_url)  # file URL, e.g.: https://ucarecdn.com/7d4e9387-26c2-4e58-840e-9254f1871c94/
+```
+
+If you don’t have the “Automatic file storing” setting enabled in your project, don’t forget to store the uploaded file:
```

Comment:
@evgkirov We've changed defaults for new users (now default is auto, and in dashboard autostore is on), so I guess we can skip this.